extension/src/paymentHandler/payment-handler.js

```diff
@@ -39,9 +41,31 @@ function _getPaymentHandlers (paymentObject) {
     handlers.push(getPaymentMethodsHandler)
   if (paymentObject.custom.fields.makePaymentRequest && !paymentObject.custom.fields.makePaymentResponse)
     handlers.push(makePaymentHandler)
+  if (paymentObject.custom.fields.makePaymentResponse
+    && paymentObject.custom.fields.submitAdditionalPaymentDetailsRequest
+    && isSubmitAdditionalPaymentDetailsRequestNew(paymentObject))
```

Comment:
what if `submitAdditionalPaymentDetailsResponse` is not empty ?

Reply:
It doesn't matter here. If request is different, we make a call.

Reply:
But I see there's a different problem here. Since we're adding some stuff to the request and then save this enriched request to interfaceInteraction, it will always be different than the one created by the shopper. I will change this.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/271/commits/d8f9e9d5f48509678153ad8ba2749d861ee46850

Reply:
I decided to put the response not empty check as you suggested as it's more consistent to our approach: https://github.com/commercetools/commercetools-adyen-integration/pull/271/commits/d8f9e9d5f48509678153ad8ba2749d861ee46850#diff-b7da223fe1867c89db8963c55099bdccR46